### ISSUE TITLE
Tabs in Dutch

### ISSUE THREAD

**Issue by kyrryl1** [pnp/custom-learning-office-365#566]
## Describe the bug

After deploying Microsoft Learning Pathway, two tabs are showing in Dutch language: Training voor Microsoft 365Stel vragen en vraag hulp.
All the other content is in english (as expected).
I am a tech support engineer and customer's tenant has the same issue.

### To Reproduce

You can check the issue by using my test account:
AdeleV@M365x816363.OnMicrosoft.com
791764Matt@

### Expected behavior

Tabs will be showing in English

### Screenshots

![image](https://user-images.githubusercontent.com/87752239/126452094-5a407ecc-fa31-4272-a321-06d91ff9039f.png)

### Learning Pathways version number

Up to date - 4.3.0.0

**Comment by jellis-thrive** [comment]
Hello @kyrryl1 

Can you please check the Browser profile settings, and M365 language profile settings, for the users experiencing this? Can we confirm they are set to English in both locations?

**Comment by dcashpeterson** [comment]
@kyrryl1 , we have seen this happen before on sites provisioned from the LookBook. Did you provision learning pathways from the LookBook or manually? See issue #539. If you provisioned it manually let us know.

**Comment by kyrryl1** [comment]
Hello Team,

Thanks for your email.

I can confirm following:


  1.  browser settings are set to English:

***@***.***D77EE3.E66AE4C0]

  1.  User's profile was set to English (Australia) but I asked customer to remove it:

***@***.***D77EE3.E66AE4C0]


  1.  User's personal site has only English as default and no other languages selected:

***@***.***D77EE4.3BB32B40]


  1.  The page's language settings are showing en-US:

***@***.***D77EE4.3BB32B40]


  1.  Microsoft 365 learning pathways language settings set to English only:

***@***.***D77EE4.4DE4C300]

Please, suggest.
Kind regards,
Kyryl

**Comment by kyrryl1** [comment]
Hi Derek,

Thanks for your reply.

Could you please confirm if the instruction for manual provisioning looks like below?

custom-learning-office-365/README.md at main * pnp/custom-learning-office-365 (github.com)<https://github.com/pnp/custom-learning-office-365/blob/main/installation/README.md>

Kind regards,
Kyryl

**Comment by dcashpeterson** [comment]
@kyrryl1 Those are the manual instructions. Did you use those instructions or did you provision the site from the LookBook?

**Comment by J04nn4** [comment]
I just used the lookbook provisioning and get TABs in Dutch. My user is set to English. But I also had it with no language for the user and still the same result. I can repro this each time I provision it via lookbook.

**Comment by dcashpeterson** [comment]
@J04nn4  thanks for letting me know. I can check things on the Learning Pathways side but I would also suggest posting the issue in the provisioning side. 

https://github.com/SharePoint/sp-provisioning-service/issues

**Comment by maxbaldt** [comment]
I just deployed it myself using the docs & lookbook: https://docs.microsoft.com/en-us/office365/customlearning/custom_provision

Same issue for me. User has no language set - is this somehow connected to the location of the tenant (west europe)?

**Comment by maxbaldt** [comment]
Oh well, I just read it all... it's https://github.com/SharePoint/sp-provisioning-service/issues/542 .. thanks & sorry for not reading everything here in the first place!

**Comment by dcashpeterson** [comment]
I will check in with the provisioning team after the summer holidays.

**Comment by SJM-SMACC** [comment]
Has anyone resolved this? I also have installed from the look book with a newly created Learning Path site and get the Dutch menus but English content. No language was selected in the Delve profile and I followed all of the steps in the Provisioning docs on the Microsoft site first validating all of the pre-reqs. All of the browsers I have used are set to English - i.e. no Dutch language settings anywhere either in the Org. or any of SharePoint or any of my system settings or browsers.

**Comment by dcashpeterson** [comment]
We are still working on this I met with the provisioning ream today and we are looking into it collectively. I will update here when it's resolved.

**Comment by maxbaldt** [comment]
Hi, I have opened this one with the docs team: https://github.com/MicrosoftDocs/OfficeDocs-CustomLearning/issues/71. Maybe it's not the root cause; @dcashpeterson can look deeper into that I guess
For me it worked after I selected en-us in delve for my admin...

**Comment by SJM-SMACC** [comment]
So it worked for you after you set Delv to English as your selected language. Did you have to wipe everything out and then redeploy the whole site?

**Comment by maxbaldt** [comment]
Redeployment was the quickest solution for me - so yes. Didn't dig too deep to find another way tbh...

**Comment by SJM-SMACC** [comment]
That worked. I removed the package from the app catalog. Deleted the site from the sites list, permanently deleted the site from the deleted sites list. Set English (United States) in the Delve profile for the Admin account doing the provisioning. Re-ran the Provisioning from the Look Book. (https://lookbook.microsoft.com/details/3df8bd55-b872-4c9d-88e3-6b2f05344239), I did choose a different site URL just to be sure (permanent deleted site URLs sometimes aren't available for 24 hours). 

If the instructions on the pre-reqs page  would just stated to set the default language to English, this would have saved me a few days of head banging. (Ensure the Tenant Admin account doesn't have a language selected) (https://docs.microsoft.com/en-us/office365/customlearning/custom_provision#turn-off-multilingual-support

Thanks for your help.

**Comment by dcashpeterson** [comment]
@maxbaldt you are correct. Sometimes setting the language to English, deleting the site, and redeploying fixes the issue and can be a viable work around. Sometimes it does not. There is an underlying issue with how the site is provisioned from the LookBook and I am working with that team to get it resolved. I will update here when I have a permanent resolution. Once we have determined the correct cause we will update the documentation as needed to reflect any changes that might need to happen.

**Comment by maxbaldt** [comment]
Great to hear! - Thanks @dcashpeterson !!
